packages/logo/composer.json

```diff
@@ -2,6 +2,7 @@
 	"name": "automattic/jetpack-logo",
 	"description": "A logo for Jetpack",
 	"type": "library",
+	"version": "1.0.0",
```

Comment:
By all means, I'd prefer not having to specify this. After all, currently we only need it so composer locally can find the package and symlink/mirror it. Let's avoid that by setting a lower minimum stability for the package (as we don't care about stability - we'll always use the local package). That can happen by specifying `@dev` as the version/stability constraint.